bot/resources/evergreen/trivia_quiz.json

```diff
@@ -269,5 +287,322 @@
       "answer": "1492",
       "info": "The explorer Christopher Columbus made four trips across the Atlantic Ocean from Spain: in 1492, 1493, 1498 and 1502. He was determined to find a direct water route west from Europe to Asia, but he never did. Instead, he stumbled upon the Americas"
     }
+  ],
+  "math": [
+    {
+      "id": 201,
+      "question": "What is the highest power of a biquadratic polynomial?",
+      "answer": "4, four"
+    },
+    {
+      "id": 202,
+      "question": "What is the formula for surface area of a sphere?",
+      "answer": "4pir^2, 4πr^2"
+    },
+    {
+      "id": 203,
+      "question": "Which theorem states that hypotenuse^2 = base^2 + height2^?",
+      "answer": "Pythagorean's, Pythagorean's theorem"
+    },
+    {
+      "id": 204,
+      "question": "Which trigonometric function is defined as hypotenuse/opposite?",
+      "answer": "cosecant, cosec, csc"
+    },
+    {
+      "id": 205,
+      "question": "Does the harmonic series converge or diverge?",
+      "answer": "diverge"
+    },
+    {
+      "id": 206,
+      "question": "How many quadrants are there in a cartesian plane?",
+      "answer": "4, four"
+    },
+    {
+      "id": 207,
+      "question": "What is the (0,0) coordinate in a cartesian plane termed as?",
+      "answer": "origin"
+    },
+    {
+      "id": 208,
+      "question": "What's the following formula that finds the area of a triangle called?",
+      "img_url": "https://wikimedia.org/api/rest_v1/media/math/render/png/d22b8566e8187542966e8d166e72e93746a1a6fc",
+      "answer": "Heron's formula"
```

Comment:
Just `heron` should also be a possible answer.

Reply:
resolved. thanks for the suggestion